docs/plugins/stonesense.rst

```diff
@@ -68,8 +68,9 @@ at least a dual core CPU to avoid slowing down your game of DF.
 
 Useful links
 ------------
+- `Report issues on Github <https://github.com/DFHack/stonesense/issues>`
 - :forums:`Official Stonesense thread <106497>` for feedback,
```

Comment:
Keeping a link to the forums is probably still a good idea, but I think the DFHack Discord is the best place for questions, requests, and feedback nowadays. Maybe add a link back to the support page as the first (or second) list item:
```md
- `support`
```

Reply:
agreed. what is the link ti the discord i should use?

Reply:
use ```
- `support`
```
where there is already information on how to get to the discord